opsdroid/core.py

```diff
@@ -114,15 +114,14 @@ def critical(self, error, code):
     @staticmethod
     def handle_async_exception(loop, context):
         """Handle exceptions from async coroutines."""
+        print("ERROR: Unhandled exception in opsdroid, exiting...")
```

Comment:
Note to reviewer: using logging here doesn't actually work, I found a cpython bug about it, but now it's lost to the sands of time.

Reply:
Ah fun! You beat me to it!